lib/NQuads.js

```diff
@@ -440,6 +446,17 @@ function _stringLiteralUnescape(s) {
 }
 
 const _iriEscapeRegex = /[\u0000-\u0020<>"{}|^`\\]/g;
+const _iriEscapeRegexMap = [];
```

Comment:
How does an actual `Map` perform?

```suggestion
const _iriEscapeRegexMap = new Map();
```

Reply:
In another conversion, we decided to try this later if needed.  It's using a 128 element array so space overhead is minimal and index lookup is likely fast.